app/bitbucket.yml

```diff
@@ -7,6 +7,7 @@ settings:
     application_hostname: test_bitbucket_instance.atlassian.com   # Bitbucket DC hostname without protocol and port e.g. test-bitbucket.atlassian.com or localhost
     application_protocol: http      # http or https
     application_port: 80            # 80, 443, 8080, 7990 etc
+    secure: True                    # Allow insecure connection, e.g. usage of Self-Signed SSL certificates
```

Comment:
Pls change the message to:
`# Set False to allow insecure connections, e.g. when using Self-Signed SSL certificate`